executor/runtime/docker/docker.go

```diff
@@ -2815,3 +2802,16 @@ func shouldClose(c io.Closer) {
 		log.Error("Could not close: ", err)
 	}
 }
+
+func (r *DockerRuntime) isPodPrivileged(pod *v1.Pod) bool {
+	if pod == nil {
+		return false
+	}
+	if pod.Spec.SecurityContext == nil {
+		return false
+	}
+	if len(pod.Spec.Containers) < 1 {
+		return false
+	}
+	return *pod.Spec.Containers[0].SecurityContext.Privileged && r.cfg.InStandaloneMode
```

Comment:
Could there be a situation where there's >1 containers, and where one of the OTHER ones is marked priveleged? (In other words does it make sense to iterate over the slice)